cmd/mysterium_client/cli/command.go

```diff
@@ -126,7 +126,7 @@ func (c *Command) connect(line string) {
 	consumerId, providerId := identities[0], identities[1]
 
 	if consumerId == "new" {
-		id, err := c.tequilapi.NewIdentity(identityDefaultPassword)
```

Comment:
But unlock passphrase should be taken from `Options.IdentityPassphrase`.
And Golang will default this option to `""`, You dont need constant for that

Reply:
There is no `Options` here - it's CLI. It should be passed in a command `identity new mypassphrase` - I've already created a subtask for that https://mysteriumnetwork.atlassian.net/browse/MYST-265. This subtask is only about renaming.

Reply:
clear